tests/lac/diagonal_matrix_02.with_trilinos=true.with_mpi=true.with_p4est=true.mpirun=7.output

```diff
@@ -0,0 +1,13 @@
+
+DEAL:2d::Testing FE_Q<2>(1)
+DEAL:2d::Ok.
+DEAL:2d::
+DEAL:2d::Testing FE_Q<2>(2)
+DEAL:2d::Ok.
+DEAL:2d::
+DEAL:3d::Testing FE_Q<3>(1)
+DEAL:3d::Ok.
+DEAL:3d::
+DEAL:3d::Testing FE_Q<3>(2)
+DEAL:3d::Ok.
+DEAL:3d::
```

Comment:
Can you remove this second test? There is no real difference between 3 and 7 processors (I suppose) on this test case since even with 7 processors there are elements on each processor. This way we keep the test suite reasonably sized. The only variant where I could see this to be useful in this setting would be if you manage to get a mesh on 7 procs where one or two procs do not own any element.